### PR TITLE
refactor: remove disco

### PR DESCRIPTION
**Pull request by mirestrepo**
This PR removes disco components, styles and font awesome pro registry.

It also centralizes a repeated back-ground color that I had in several components

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -41,7 +41,6 @@ jobs:
 
       - name: Install dependencies 👨🏻‍💻
         run: |
-          npm config set "//npm.pkg.github.com/:_authToken" ${{ secrets.GH_TOKEN_DISCO }}
           yarn
```
Comment (mcmcgrath13) [inline .github/workflows/ci.yml:44]:
```suggestion
```

**Review by mcmcgrath13 — APPROVED** [review]
so much deleting!!!

File: components/Navbar.vue [hunk 2]
```diff
@@ -1,38 +1,35 @@
...
-    </DNavbar>
-  </div>
+  <b-navbar>
+    <template v-slot:brand>
+      <a
+        class="navbar-item navbar-header-end"
+        href="https://brown.edu"
+        aria-label="Brown's Home"
+      >
+        <BrownLogo />
```
[the remark this answers is not in the document]
Comment (mirestrepo) [reply components/Navbar.vue:9]: I don't think so. I won't be using the logo more than once, but if I could bring them back in future if needed. The Disco log was a bit more generalizable for good reasons.

**Review by eldu — APPROVED** [review]
Looks good to me! Lots of clean up with the removal of the disco components and the font awesome pro registry! Always a good time deleting stuff !

**Pull request merged**